README.md

```diff
@@ -138,7 +148,7 @@ Single e2e tests can be run with `yarn test:e2e:single test/e2e/tests/TEST_NAME.
 ```
 
 For example, to run the `account-details` tests using Chrome, with debug logging and with the browser set to remain open upon failure, you would use:
-`yarn test:e2e:single test/e2e/tests/account-details.spec.js --browser=chrome --debug --leave-running`
+`yarn test:e2e:single test/e2e/tests/account-menu/account-details.spec.js --browser=chrome --debug --leave-running`
```

Comment:
Shall we add `multichain` one as well, as `MULTICHAIN=1 yarn test:e2e:chrome xxx`

Reply:
Thanks Danica, that's a great suggestion. I think it's more accurate to enable feature flag when we create test build. I pushed a new section about how to run e2e tests against feature flag. Please check the new commit: 
https://github.com/MetaMask/metamask-extension/pull/23309/commits/6ec8bc8ea4f1bd22325696e8733360e6292bcc6c